### ISSUE TITLE
Zero network in Android (Samsung Galaxy Tab S6)

### ISSUE THREAD

**Issue by ErunamoJAZZ** [ppy/osu#24827]
### Type

Game behaviour

### Bug description

Hello, I have a Galaxy Tab S6, when I try to run osu!, this works just fine and smooth, except it is unavailable to connect to internet (it looks related to #20609, #17499 and #17858 btw).

Exact model of my Samsung Galaxy Tab S6 is SM-T867U, but I think this is related to the fact this is a Samsung device, and a little old model maybe. I don't have any other Samsung device to test it right now tho.

- Android 12 (last available update from August 8th, 2023)
- One UI 4.1.1

I just opened the game and tried to "Get the osu! tutorial" at the First-run setup, but I got a timeout. Also tried to login, but it don't work neither.

### Screenshots or videos

![img1](https://cdn.discordapp.com/attachments/1148088821657907281/1152435014416482467/IMG20230915213924.jpg)
![img2](https://cdn.discordapp.com/attachments/1148088821657907281/1152435014714261584/IMG20230915213944.jpg)
![img3](https://cdn.discordapp.com/attachments/1148088821657907281/1152435014991097976/IMG20230915214038.jpg)
![img4](https://cdn.discordapp.com/attachments/1148088821657907281/1152435015293091921/IMG20230915214051.jpg)
![img5](https://cdn.discordapp.com/attachments/1148088821657907281/1152435015536345219/IMG20230915214112.jpg)
![img6](https://cdn.discordapp.com/attachments/1148088821657907281/1152435015800598628/IMG20230915214207.jpg)
![img7](https://cdn.discordapp.com/attachments/1148088821657907281/1152435016119369748/IMG20230915214221.jpg)
![img8](https://cdn.discordapp.com/attachments/1148088821657907281/1152435016438120468/IMG20230915214240.jpg)

### Version

2023.914.0

### Logs

```
2023-09-16 02:38:58 [verbose]: ----------------------------------------------------------
2023-09-16 02:38:58 [verbose]: runtime Log for u0_a359 (LogLevel: Verbose)
2023-09-16 02:38:58 [verbose]: Running  2023.914.0.0 on .NET 6.0.22
2023-09-16 02:38:58 [verbose]: Environment: Android (Unix 32.0.0.0), 8 cores
2023-09-16 02:38:58 [verbose]: ----------------------------------------------------------
2023-09-16 02:38:58 [verbose]: 🖼️ Configuration renderer choice: Automatic
2023-09-16 02:38:58 [verbose]: 🖼️ Renderer fallback order: [ OpenGL (Legacy), OpenGL, Vulkan ]
2023-09-16 02:38:58 [verbose]: 🖼️ Initialising "GL" renderer with "OpenGL" surface
2023-09-16 02:38:58 [verbose]: GL Initialized
2023-09-16 02:38:58 [verbose]: GL Version:                 OpenGL ES 3.2 V@0502.0 (GIT@390f4e9c29, Ie058859f22, 1657949000) (Date:07/15/22)
2023-09-16 02:38:58 [verbose]: GL Renderer:                Adreno (TM) 640
2023-09-16 02:38:58 [verbose]: GL Shader Language version: OpenGL ES GLSL ES 3.20
2023-09-16 02:38:58 [verbose]: GL Vendor:                  Qualcomm
2023-09-16 02:38:58 [verbose]: GL Extensions:              GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_EGL_sync GL_OES_vertex_half_float GL_OES_framebuffer_object GL_OES_rgb8_rgba8 GL_OES_compressed_ETC1_RGB8_texture GL_AMD_compressed_ATC_texture GL_KHR_texture_compression_astc_ldr GL_KHR_texture_compression_astc_hdr GL_OES_texture_compression_astc GL_OES_texture_npot GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_read_format_bgra GL_OES_texture_3D GL_EXT_color_buffer_float GL_EXT_color_buffer_half_float GL_QCOM_alpha_test GL_OES_depth24 GL_OES_packed_depth_stencil GL_OES_depth_texture GL_OES_depth_texture_cube_map GL_EXT_sRGB GL_OES_texture_float GL_OES_texture_float_linear GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_EXT_texture_type_2_10_10_10_REV GL_EXT_texture_sRGB_decode GL_EXT_texture_format_sRGB_override GL_OES_element_index_uint GL_EXT_copy_image GL_EXT_geometry_shader GL_EXT_tessellation_shader GL_OES_texture_stencil8 GL_EXT_shader_io_blocks GL_OES_shader_image_atomic GL_OES_sample_variables GL_EXT_texture_border_clamp GL_EXT_EGL_image_external_wrap_modes GL_EXT_multisampled_render_to_texture GL_EXT_multisampled_render_to_texture2 GL_OES_shader_multisample_interpolation GL_EXT_texture_cube_map_array GL_EXT_draw_buffers_indexed GL_EXT_gpu_shader5 GL_EXT_robustness GL_EXT_texture_buffer GL_EXT_shader_framebuffer_fetch GL_ARM_shader_framebuffer_fetch_depth_stencil GL_OES_texture_storage_multisample_2d_array GL_OES_sample_shading GL_OES_get_program_binary GL_EXT_debug_label GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_QCOM_tiled_rendering GL_ANDROID_extension_pack_es31a GL_EXT_primitive_bounding_box GL_OES_standard_derivatives GL_OES_vertex_array_object GL_EXT_disjoint_timer_query GL_KHR_debug GL_EXT_YUV_target GL_EXT_sRGB_write_control GL_EXT_texture_norm16 GL_EXT_discard_framebuffer GL_OES_surfaceless_context GL_OVR_multiview GL_OVR_multiview2 GL_EXT_texture_sRGB_R8 GL_KHR_no_error GL_EXT_debug_marker GL_OES_EGL_image_external_essl3 GL_OVR_multiview_multisampled_render_to_texture GL_EXT_buffer_storage GL_EXT_external_buffer GL_EXT_blit_framebuffer_params GL_EXT_clip_cull_distance GL_EXT_protected_textures GL_EXT_shader_non_constant_global_initializers GL_QCOM_texture_foveated GL_QCOM_texture_foveated_subsampled_layout GL_QCOM_shader_framebuffer_fetch_noncoherent GL_QCOM_shader_framebuffer_fetch_rate GL_EXT_memory_object GL_EXT_memory_object_fd GL_EXT_EGL_image_array GL_NV_shader_noperspective_interpolation GL_KHR_robust_buffer_access_behavior GL_EXT_EGL_image_storage GL_EXT_blend_func_extended GL_EXT_clip_control GL_OES_texture_view GL_EXT_fragment_invocation_density GL_QCOM_motion_estimation GL_QCOM_validate_shader_binary GL_QCOM_YUV_texture_gather
2023-09-16 02:38:58 [verbose]: UseStructuredBuffers: False
2023-09-16 02:38:58 [verbose]: 🖼️ Renderer initialised!
2023-09-16 02:38:58 [verbose]: Host execution state changed to Running
2023-09-16 02:38:58 [verbose]: Execution mode changed to SingleThread
2023-09-16 02:38:58 [verbose]: Beginning realm file store cleanup
2023-09-16 02:38:58 [verbose]: Finished realm file store cleanup (0 of 2 deleted)
2023-09-16 02:38:59 [verbose]: Opened realm "/storage/emulated/0/Android/data/sh.ppy.osulazer/files/client.realm" at version 34
2023-09-16 02:39:00 [verbose]: Loading FPSCounter...
2023-09-16 02:39:00 [verbose]: 🔈 BASS initialised
2023-09-16 02:39:00 [verbose]: BASS version:           2.4.16.7
2023-09-16 02:39:00 [verbose]: BASS FX version:        2.4.12.6
2023-09-16 02:39:00 [verbose]: BASS MIX version:       2.4.11.1
2023-09-16 02:39:00 [verbose]: Device:                 Default
2023-09-16 02:39:00 [verbose]: Driver:
2023-09-16 02:39:00 [verbose]: Update period:          5 ms
2023-09-16 02:39:00 [verbose]: Device buffer length:   10 ms
2023-09-16 02:39:00 [verbose]: Playback buffer length: 100 ms
2023-09-16 02:39:00 [verbose]: Loaded FPSCounter!
2023-09-16 02:39:00 [verbose]: Loading VersionManager...
2023-09-16 02:39:00 [verbose]: Loaded VersionManager!
2023-09-16 02:39:00 [verbose]: Loading OsuLogo...
2023-09-16 02:39:00 [verbose]: Loaded OsuLogo!
2023-09-16 02:39:00 [verbose]: Loading Toolbar...
2023-09-16 02:39:00 [verbose]: 📺 OsuScreenStack#111(depth:1) loading Loader#126
2023-09-16 02:39:00 [verbose]: 📺 OsuScreenStack#111(depth:1) entered Loader#126
2023-09-16 02:39:01 [verbose]: 📺 OsuScreenStack#111(depth:1) suspended Loader#126 (waiting on Disclaimer#-13)
2023-09-16 02:39:01 [verbose]: 📺 OsuScreenStack#111(depth:2) entered Disclaimer#-13
2023-09-16 02:39:01 [verbose]: Loaded Toolbar!
2023-09-16 02:39:01 [verbose]: Loading VolumeOverlay...
2023-09-16 02:39:01 [verbose]: Loaded VolumeOverlay!
2023-09-16 02:39:01 [verbose]: Loading OnScreenDisplay...
2023-09-16 02:39:01 [verbose]: Loaded OnScreenDisplay!
2023-09-16 02:39:01 [verbose]: Loading NotificationOverlay...
2023-09-16 02:39:01 [verbose]: Loaded NotificationOverlay!
2023-09-16 02:39:01 [verbose]: Loading LegacyImportManager...
2023-09-16 02:39:01 [verbose]: Loaded LegacyImportManager!
2023-09-16 02:39:01 [verbose]: Loading ScreenshotManager...
2023-09-16 02:39:01 [verbose]: Loaded ScreenshotManager!
2023-09-16 02:39:01 [verbose]: Loading SimpleUpdateManager...
2023-09-16 02:39:01 [verbose]: Loaded SimpleUpdateManager!
2023-09-16 02:39:01 [verbose]: Loading FirstRunSetupOverlay...
2023-09-16 02:39:01 [verbose]: Loaded FirstRunSetupOverlay!
2023-09-16 02:39:01 [verbose]: Loading ManageCollectionsDialog...
2023-09-16 02:39:01 [verbose]: Loaded ManageCollectionsDialog!
2023-09-16 02:39:01 [verbose]: Loading BeatmapListingOverlay...
2023-09-16 02:39:01 [verbose]: Loaded BeatmapListingOverlay!
2023-09-16 02:39:01 [verbose]: Loading DashboardOverlay...
2023-09-16 02:39:01 [verbose]: Loaded DashboardOverlay!
2023-09-16 02:39:01 [verbose]: Loading NewsOverlay...
2023-09-16 02:39:01 [verbose]: Loaded NewsOverlay!
2023-09-16 02:39:01 [verbose]: Loading RankingsOverlay...
2023-09-16 02:39:02 [verbose]: Loaded RankingsOverlay!
2023-09-16 02:39:02 [verbose]: Loading ChannelManager...
2023-09-16 02:39:02 [verbose]: Loaded ChannelManager!
2023-09-16 02:39:02 [verbose]: Loading ChatOverlay...
2023-09-16 02:39:02 [verbose]: Loaded ChatOverlay!
2023-09-16 02:39:02 [verbose]: Loading MessageNotifier...
2023-09-16 02:39:02 [verbose]: Loaded MessageNotifier!
2023-09-16 02:39:02 [verbose]: Loading SettingsOverlay...
2023-09-16 02:39:02 [verbose]: Loaded SettingsOverlay!
2023-09-16 02:39:02 [verbose]: Loading ChangelogOverlay...
2023-09-16 02:39:02 [verbose]: Loaded ChangelogOverlay!
2023-09-16 02:39:02 [verbose]: Loading UserProfileOverlay...
2023-09-16 02:39:02 [verbose]: Loaded UserProfileOverlay!
2023-09-16 02:39:02 [verbose]: Loading BeatmapSetOverlay...
2023-09-16 02:39:02 [verbose]: Loaded BeatmapSetOverlay!
2023-09-16 02:39:02 [verbose]: Loading WikiOverlay...
2023-09-16 02:39:02 [verbose]: Loaded WikiOverlay!
2023-09-16 02:39:02 [verbose]: Loading SkinEditorOverlay...
2023-09-16 02:39:02 [verbose]: Loaded SkinEditorOverlay!
2023-09-16 02:39:02 [verbose]: Loading LoginOverlay...
2023-09-16 02:39:02 [verbose]: Loaded LoginOverlay!
2023-09-16 02:39:02 [verbose]: Loading NowPlayingOverlay...
2023-09-16 02:39:02 [verbose]: Loaded NowPlayingOverlay!
2023-09-16 02:39:02 [verbose]: Loading AccountCreationOverlay...
2023-09-16 02:39:02 [verbose]: 📺 ScreenStack#-85(depth:1) scheduling push ScreenWelcome#-40
2023-09-16 02:39:02 [verbose]: Loaded AccountCreationOverlay!
2023-09-16 02:39:02 [verbose]: Loading DialogOverlay...
2023-09-16 02:39:02 [verbose]: Loaded DialogOverlay!
2023-09-16 02:39:02 [verbose]: Loading HighPerformanceSession...
2023-09-16 02:39:02 [verbose]: Loaded HighPerformanceSession!
2023-09-16 02:39:02 [verbose]: Loading BackgroundDataStoreProcessor...
2023-09-16 02:39:02 [verbose]: Loaded BackgroundDataStoreProcessor!
2023-09-16 02:39:02 [verbose]: Beginning background data store processing..
2023-09-16 02:39:02 [verbose]: Querying for beatmap sets to reprocess...
2023-09-16 02:39:02 [verbose]: Found 0 beatmap sets which require reprocessing.
2023-09-16 02:39:02 [verbose]: Querying for scores to reprocess...
2023-09-16 02:39:02 [verbose]: Found 0 scores which require reprocessing.
2023-09-16 02:39:02 [verbose]: Querying for scores that need total score conversion...
2023-09-16 02:39:02 [verbose]: Found 0 scores which require total score conversion.
2023-09-16 02:39:02 [verbose]: Finished background data store processing!
2023-09-16 02:39:07 [verbose]: 📺 OsuScreenStack#111(depth:2) suspended Disclaimer#-13 (waiting on IntroTriangles#172)
2023-09-16 02:39:07 [verbose]: 📺 OsuScreenStack#111(depth:3) entered IntroTriangles#172
2023-09-16 02:39:07 [verbose]: 📺 BackgroundScreenStack#-61(depth:1) loading BackgroundScreenDefault#-89
2023-09-16 02:39:07 [verbose]: 🌅 Global background change queued
2023-09-16 02:39:07 [verbose]: Game-wide working beatmap updated to cYsmix - triangles (peppy) [peppy]
2023-09-16 02:39:07 [verbose]: 📺 BackgroundScreenStack#-61(depth:1) entered BackgroundScreenDefault#-89
2023-09-16 02:39:07 [verbose]: decoupled ruleset transferred ("" -> "osu!")
2023-09-16 02:39:07 [verbose]: 🌅 Global background loading
2023-09-16 02:39:10 [verbose]: 📺 OsuScreenStack#111(depth:3) suspended IntroTriangles#172 (waiting on MainMenu#-10)
2023-09-16 02:39:10 [verbose]: 📺 OsuScreenStack#111(depth:4) entered MainMenu#-10
2023-09-16 02:39:13 [verbose]: ButtonSystem's state changed from Initial to TopLevel
2023-09-16 02:39:17 [verbose]: ButtonSystem's state changed from TopLevel to Play
2023-09-16 02:39:18 [verbose]: 📺 OsuScreenStack#111(depth:4) suspended MainMenu#-10 (waiting on PlaySongSelect#196)
2023-09-16 02:39:18 [verbose]: ButtonSystem's state changed from Play to EnteringMode
2023-09-16 02:39:18 [verbose]: 📺 OsuScreenStack#111(depth:5) entered PlaySongSelect#196
2023-09-16 02:39:18 [verbose]: 📺 BackgroundScreenStack#-61(depth:2) loading BackgroundScreenBeatmap#569
2023-09-16 02:39:19 [verbose]: 📺 BackgroundScreenStack#-61(depth:2) suspended BackgroundScreenDefault#-89 (waiting on BackgroundScreenBeatmap#569)
2023-09-16 02:39:19 [verbose]: 📺 BackgroundScreenStack#-61(depth:2) entered BackgroundScreenBeatmap#569
2023-09-16 02:39:19 [verbose]: Song select updating selection with beatmap:null ruleset:osu
2023-09-16 02:39:19 [verbose]: Song select changing beatmap from "cYsmix - triangles (peppy) [peppy]" to "null"
2023-09-16 02:39:19 [verbose]: Game-wide working beatmap updated to please load a beatmap! - no beatmaps available!
2023-09-16 02:39:19 [verbose]: Song select decided to ensurePlayingSelected
2023-09-16 02:39:19 [verbose]: 🌅 Global background change queued
2023-09-16 02:39:26 [verbose]: 📺 BackgroundScreenStack#-61(depth:1) exit from BackgroundScreenBeatmap#569
2023-09-16 02:39:26 [verbose]: 📺 BackgroundScreenStack#-61(depth:1) resume to BackgroundScreenDefault#-89
2023-09-16 02:39:26 [verbose]: 📺 OsuScreenStack#111(depth:4) exit from PlaySongSelect#196
2023-09-16 02:39:26 [verbose]: 📺 OsuScreenStack#111(depth:4) resume to MainMenu#-10
2023-09-16 02:39:26 [verbose]: MusicController skipping next track to EnsurePlayingSomething
2023-09-16 02:39:26 [verbose]: decoupled ruleset transferred ("" -> "osu!")
2023-09-16 02:39:26 [verbose]: Game-wide working beatmap updated to cYsmix - triangles (peppy) [peppy]
2023-09-16 02:39:26 [verbose]: 🌅 Global background change queued
2023-09-16 02:39:26 [verbose]: 🌅 Global background change queued
2023-09-16 02:39:26 [verbose]: ButtonSystem's state changed from EnteringMode to TopLevel
2023-09-16 02:39:26 [verbose]: ButtonSystem's state changed from TopLevel to Initial
2023-09-16 02:39:26 [verbose]: 📺 ScreenStack#528(depth:1) loading ScreenWelcome#-18
2023-09-16 02:39:26 [verbose]: 📺 ScreenStack#528(depth:1) entered ScreenWelcome#-18
2023-09-16 02:39:26 [verbose]: 🌅 Global background loading
2023-09-16 02:39:28 [verbose]: 📺 ScreenStack#528(depth:1) suspended ScreenWelcome#-18 (waiting on ScreenUIScale#-14)
2023-09-16 02:39:28 [verbose]: 📺 ScreenStack#528(depth:2) loading ScreenUIScale#-14
2023-09-16 02:39:28 [verbose]: decoupled ruleset transferred ("" -> "osu!")
2023-09-16 02:39:28 [verbose]: 📺 OsuScreenStack#212(depth:1) entered ScreenUIScale+NestedSongSelect#-12
2023-09-16 02:39:28 [verbose]: 📺 ScreenStack#528(depth:2) entered ScreenUIScale#-14
2023-09-16 02:39:28 [verbose]: 📺 BackgroundScreenStack#-16(depth:1) loading BackgroundScreenBeatmap#746
2023-09-16 02:39:28 [verbose]: 📺 BackgroundScreenStack#-16(depth:1) entered BackgroundScreenBeatmap#746
2023-09-16 02:39:28 [verbose]: Song select updating selection with beatmap:null ruleset:osu
2023-09-16 02:39:28 [verbose]: Song select changing beatmap from "please load a beatmap! - no beatmaps available!" to "null"
2023-09-16 02:39:29 [verbose]: 📺 ScreenStack#528(depth:2) suspended ScreenUIScale#-14 (waiting on ScreenBeatmaps#106)
2023-09-16 02:39:29 [verbose]: 📺 ScreenStack#528(depth:3) loading ScreenBeatmaps#106
2023-09-16 02:39:29 [verbose]: 📺 ScreenStack#528(depth:3) entered ScreenBeatmaps#106
2023-09-16 02:40:46 [error]: Beatmap download failed!
2023-09-16 02:40:46 [error]: System.Net.WebException: Request to https://assets.ppy.sh/client-resources/bundled/1011011.osz timed out after 60 seconds idle (read 0 bytes, retried 0 times).
2023-09-16 02:40:46 [error]: at osu.Framework.IO.Network.WebRequest.Complete(Exception e)
2023-09-16 02:40:46 [error]: at osu.Framework.IO.Network.FileWebRequest.Complete(Exception e)
2023-09-16 02:40:46 [error]: at osu.Framework.IO.Network.WebRequest.internalPerform(CancellationToken cancellationToken)
2023-09-16 02:40:46 [error]: at osu.Framework.IO.Network.WebRequest.PerformAsync(CancellationToken cancellationToken)
2023-09-16 02:40:46 [error]: at osu.Framework.Extensions.ExceptionExtensions.ExceptionExtensions.Rethrow(Exception exception)
2023-09-16 02:40:46 [error]: at osu.Framework.Extensions.ExceptionExtensions.ExceptionExtensions.RethrowAsSingular(AggregateException aggregateException)
2023-09-16 02:40:46 [error]: at osu.Framework.IO.Network.WebRequest.Perform()
2023-09-16 02:40:46 [error]: at osu.Game.Online.API.APIRequest.Perform(IAPIProvider api)
2023-09-16 02:40:46 [error]: at osu.Game.Online.API.APIAccess.Perform(APIRequest request)
2023-09-16 02:40:46 [verbose]: ⚠️ Beatmap download failed!
2023-09-16 02:40:46 [verbose]: 
2023-09-16 02:40:46 [verbose]: This error has been automatically reported to the devs.
2023-09-16 02:40:54 [verbose]: ⚠️ Click to resume first-run setup at any point
2023-09-16 02:40:57 [verbose]: ButtonSystem's state changed from Initial to TopLevel
2023-09-16 02:40:58 [verbose]: Focus contention triggered by NotificationOverlay.
2023-09-16 02:41:16 [verbose]: Focus contention triggered by LoginOverlay.
2023-09-16 02:42:16 [verbose]: 🌅 Global background change queued
2023-09-16 02:42:16 [verbose]: 🌅 Global background loading
2023-09-16 02:42:36 [verbose]: 🌅 Global background change queued
2023-09-16 02:42:36 [verbose]: 🌅 Global background loading
```

```
2023-09-16 02:17:21 [verbose]: ----------------------------------------------------------
2023-09-16 02:17:21 [verbose]: network Log for u0_a359 (LogLevel: Verbose)
2023-09-16 02:17:21 [verbose]: Running  2023.914.0.0 on .NET 6.0.22
2023-09-16 02:17:21 [verbose]: Environment: Android (Unix 32.0.0.0), 8 cores
2023-09-16 02:17:21 [verbose]: ----------------------------------------------------------
2023-09-16 02:17:21 [verbose]: Request to https://api.github.com/repos/ppy/osu/releases/latest successfully completed!
2023-09-16 02:17:38 [verbose]: Request to https://assets.ppy.sh/client-resources/online.db.bz2?20230916 failed with System.Net.WebException: Request to https://assets.ppy.sh/client-resources/online.db.bz2?20230916 timed out after 20 seconds idle (read 0 bytes, retried 0 times). (retrying 1/1).
2023-09-16 02:17:58 [verbose]: Request to https://assets.ppy.sh/client-resources/online.db.bz2?20230916 failed with System.Net.WebException: Request to https://assets.ppy.sh/client-resources/online.db.bz2?20230916 timed out after 20 seconds idle (read 0 bytes, retried 1 times)..
2023-09-16 02:18:05 [verbose]: Request to https://lazer.ppy.sh/oauth/token failed with System.Net.WebException: Request to https://lazer.ppy.sh/oauth/token timed out after 10 seconds idle (read 0 bytes, retried 0 times). (retrying 1/1).
2023-09-16 02:18:15 [verbose]: Request to https://lazer.ppy.sh/oauth/token failed with System.Net.WebException: Request to https://lazer.ppy.sh/oauth/token timed out after 9 seconds idle (read 0 bytes, retried 1 times)..
2023-09-16 02:18:15 [verbose]: Login failed for username ErunamoJAZZ (Request to https://lazer.ppy.sh/oauth/token timed out after 9 seconds idle (read 0 bytes, retried 1 times).)!
2023-09-16 02:19:08 [verbose]: Performing request osu.Game.Beatmaps.Drawables.BundledBeatmapDownloader+BundledBeatmapModelDownloader+BundledBeatmapDownloadRequest
2023-09-16 02:20:08 [verbose]: Request to https://assets.ppy.sh/client-resources/bundled/1011011.osz failed with System.Net.WebException: Request to https://assets.ppy.sh/client-resources/bundled/1011011.osz timed out after 60 seconds idle (read 0 bytes, retried 0 times)..
2023-09-16 02:20:08 [verbose]: Failing request osu.Game.Beatmaps.Drawables.BundledBeatmapDownloader+BundledBeatmapModelDownloader+BundledBeatmapDownloadRequest (System.Net.WebException: Request to https://assets.ppy.sh/client-resources/bundled/1011011.osz timed out after 60 seconds idle (read 0 bytes, retried 0 times).)
```

```
2023-09-16 02:17:58 [verbose]: ----------------------------------------------------------
2023-09-16 02:17:58 [verbose]: database Log for u0_a359 (LogLevel: Verbose)
2023-09-16 02:17:58 [verbose]: Running  2023.914.0.0 on .NET 6.0.22
2023-09-16 02:17:58 [verbose]: Environment: Android (Unix 32.0.0.0), 8 cores
2023-09-16 02:17:58 [verbose]: ----------------------------------------------------------
2023-09-16 02:17:58 [verbose]: BeatmapUpdaterMetadataLookup's online cache download failed: System.Net.WebException: Request to https://assets.ppy.sh/client-resources/online.db.bz2?20230916 timed out after 20 seconds idle (read 0 bytes, retried 1 times).
```

**Comment by bdach** [comment]
> it looks related to #20609, #17499 and #17858 btw

This looks like a duplicate of at least two of these. Closing as such.

**Comment by ErunamoJAZZ** [comment]
but then what of those issues would be the main issue?, if I get more info where must I attach it?

**Comment by bdach** [comment]
https://github.com/ppy/osu/issues/17499, probably
